tests/ui/dbg_macro/dbg_macro.rs

```diff
@@ -1,5 +1,5 @@
 #![warn(clippy::dbg_macro)]
-#![allow(clippy::unnecessary_operation, clippy::no_effect)]
+#![allow(clippy::unnecessary_operation, clippy::no_effect, clippy::unit_arg)]
```

Comment:
It now emits a warning on this line
https://github.com/rust-lang/rust-clippy/blob/c771204ffb6422de325150427a74e72577a3d37c/tests/ui/dbg_macro/dbg_macro.rs#L53

which looks like a true positive to me